persistence/src/main/scala/com/mesosphere/usi/persistence/repository/Repository.scala

```diff
@@ -0,0 +1,41 @@
+package com.mesosphere.usi.persistence.repository
+
+import com.mesosphere.usi.core.models.{PodId, PodRecord, ReservationId, ReservationRecord}
+
+import scala.concurrent.Future
+
+/**
+  * Base trait for storing records which are used by the Scheduler.
+  *
+  */
+trait Repository {
```

Comment:
should there maybe be a separate repository for pod records and reservation records? I always considered repository pattern as being per entity...